script-library/desktop-lite-debian.sh

```diff
@@ -94,12 +94,12 @@ apt-get-update-if-needed
 
 # On older Ubuntu, Tilix is in a PPA. On Debian Strech, its in backports
 if [[ -z $(apt-cache --names-only search ^tilix$) ]]; then
-    apt-get install -y --no-install-recommends lsb-release
-    if [ "$(lsb_release -is)" = "Ubuntu" ]; then
+    . /etc/os-release
+    if [ "${ID}" = "ubuntu" ]; then
         apt-get install -y --no-install-recommends apt-transport-https software-properties-common
         add-apt-repository -y ppa:webupd8team/terminix
```

Comment:
`ppa:webupd8team/terminix` is a legit ppa we want to pull from?

Reply:
Yep. This part is not new. It only kicks in for Ubuntu 16.04.  18.04+ have it in the main repo.